src/mats_l1_processing/photometer.py

```diff
@@ -0,0 +1,93 @@
+    pmAband_Tpd = np.NaN * np.ones_like(PM2_Tpd_bit) # Temperature of the A-band photometer photodiode
+    pmAband_Tif = np.NaN * np.ones_like(PM2_Tif_bit) # Temperature of the A-band photometer filter
+
+    for ij in range(0, len(PM1_Tpd_bit)-1):
+        index11 =  np.where(bitar == round(PM1_Tpd_bit[ij],1))
+        pmBkg_Tpd[ij] = TempFM1pd_raw[index11]
+        index12 =  np.where(bitar == round(PM1_Tif_bit[ij],1))
+        pmBkg_Tif[ij] = TempFM1if_raw[index12]
+        index13 =  np.where(bitar == round(PM2_Tpd_bit[ij],1))
+        pmAband_Tpd[ij] = TempFM2pd_raw[index13]
+        index14 =  np.where(bitar == round(PM2_Tif_bit[ij],1))
+        pmAband_Tif[ij] = TempFM2if_raw[index14]
+
+# ==============================
+# Change raw photometer data to photons cm-2 str-1 s-1
+#   define pmBkg_Sig, pmAband_Sig
+    pmBkg_Sig = np.ones_like(PM1_Sig_bit)   # Background photometer signal
+    pmAband_Sig = np.ones_like(PM2_Sig_bit) # A-band photometer signal
+    #ijk1=0
+    #ijk2=0
+
+    for ik in range(0, len(PM1_Sig_bit)-1):
+        index21 = np.where(Temperatur == round(pmBkg_Tpd[ik],1)) #OMC 2023.04.04: Shoud interpolation be used insted of lookup table?
+        index22 = np.where(Temperatur == round(pmAband_Tpd[ik],1)) #OMC 2023.04.04: Shoud interpolation be used insted of lookup table?
+    
+        index23 = np.where(bitar == round(PM1_Sig_bit[ik],1)) #OMC 2023.04.04: Shoud interpolation be used insted of lookup table?
+        if index23[1].size == 0:
+            pmBkg_Sig[ik] = np.NaN
+            #ijk1=ijk1+1
+        else:
+            pmBkg_Sig[ik] = SignFM1_Rad_raw[index21[1], index23[1]]
+    
+        index24 = np.where(bitar == round(PM2_Sig_bit[ik],1)) #OMC 2023.04.04: Shoud interpolation be used insted of lookup table?
+        if index24[1].size == 0:
+            pmAband_Sig[ik] = np.NaN
+            #ijk2=ijk2+1
+        else:
+            pmAband_Sig[ik] = SignFM2_Rad_raw[index22[1], index24[1]]
+
+# End of calibration
+# ===============================================================================================================
+# Modify dataframe before returning it
+#   Remove raw photometer data from the df dataframe
+    df.drop(df.iloc[:, 12:24], inplace=True, axis=1)
+
+#   Add calibared data to the df dataframe
+    df["pmAband_Sig"] = pmAband_Sig # A-band photometer signal [photons cm-2 str-1 s-1]
+    df["pmAband_Tpd"] = pmAband_Tpd # Temperature of the A-band photometer photodiode [°C]
+    df["pmAband_Tif"] = pmAband_Tif # Temperature of the A-band photometer interference filter [°C]
+    df["pmBkg_Sig"] = pmBkg_Sig # Background photometer signal [photons cm-2 str-1 s-1]
+    df["pmBkg_Tpd"] = pmBkg_Tpd # Temperature of the background photometer photodiode [°C]
+    df["pmBkg_Tif"] = pmBkg_Tif # Temperature of the background photometer interference filter [°C]
+    df["pm_texp"] = pm_texp # The photometer exposure time [s]
```

Comment:
Note that the corresponding attribute for CCD is called `TEXPMS`, all caps and in milli-seconds. Consider if you want to try to keep it uniform.

Reply:
what do you say to this @jonashedin

Reply:
Ok, yes why not try to keep it uniform, please change as suggested